cmd/vault-staging-k8s-secret/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/teamsnap/vault-key/pkg/k8s"
+	"github.com/teamsnap/vault-key/pkg/vault"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// var env = map[string]map[string]string{}
+
+func main() {
+	cfg := zap.Config{
+		Level:            zap.NewAtomicLevelAt(zapcore.DebugLevel),
+		Encoding:         "console",
+		EncoderConfig:    zap.NewDevelopmentEncoderConfig(),
+		OutputPaths:      []string{"stderr"},
+		ErrorOutputPaths: []string{"stderr"},
+	}
+
+	logger, err := cfg.Build()
+	if err != nil {
+		panic(err)
+	}
+	cfg.Level.SetLevel(zap.ErrorLevel)
+
+	run(context.Background(), logger)
+}
+
+func run(ctx context.Context, lgr *zap.Logger) {
+	defaultEngine := os.Getenv("VAULT_SECRET")
```

Comment:
From a testability standpoint, how do you feel about passing in this (and the other envvars) via a config object?